prow/plugins/blunderbuss/blunderbuss_test.go

```diff
@@ -593,21 +595,35 @@ func TestHandlePullRequest(t *testing.T) {
 			filesChanged:  []string{"a.go"},
 			reviewerCount: 1,
 		},
+		{
+			name:         "draft pr opened, ignore drafts",
+			action:       github.PullRequestActionOpened,
+			draft:        true,
+			ignoreDrafts: true,
+		},
```

Comment:
If we set `draft: false` here does the test fail? If it doesn't please update it so that we ensure we're testing the desired behavior.

Reply:
@cjwagner no, the test won't fail. I've updated it and added second test case when Opened PR is not a Draft. I hope this resolves the topic.